### DISCUSSION TITLE
Highlighting Keywords

### DISCUSSION THREAD

**Discussion by DesireeFSc** [asreview/asreview#1243] [Q&A]
Hello,
I am new to AsReview and am planning to use it for my systematic review. In other screening platforms, there is often the option to highlight keywords in the abstract which makes it a lot faster to evaluate if the abstract is relevant. Would this also be possible in AsReview?

**Comment by Rensvandeschoot** [comment]
Yes! See the discussion in this thread: https://github.com/asreview/asreview/discussions/995

**Reply by DesireeFSc** [reply]
Ah, nice! Thank you!

**Comment by J535D165** [comment]
There are very useful external tools for keyword highlighting. Usually, they are browser extensions. 

As so many of these nice tools exist, we never implemented this into ASReview LAB. 

Example with Multi Highlight for Chrome

<img width="1439" alt="image" src="https://user-images.githubusercontent.com/12981139/194867360-e45e9552-b6bc-4bdb-8783-a6d5f24cc517.png">

**Reply by DesireeFSc** [reply]
Thank you!
